Add Prism Launcher to the Install page (#80)
QuiltMC/quiltmc.org: src/pages/en/install/prismlauncher.md

---
title: "Install: Prism Launcher"
permalink: /install/prismlauncher/

layout: /src/layouts/InstallPage.astro
platformKey: platform-prismlauncher
website: https://prismlauncher.org
---

> Prism Launcher is a launcher designed to allow users to easily manage multiple installations of Minecraft at once as well as provide all the power tools that an advanced user would need in order to manage their instances.

Reviewer: I know it is likely just a copy paste from the previous page, but this sentence is waaaaaaay too heavy.
```suggestion
The Prism Launcher is designed to allow users to manage multiple installations of Minecraft at once easily, and provide the power tools an advanced user would need.
```
